kubernetes/templates/4-alertmanager.yaml

```diff
@@ -32,7 +32,7 @@ spec:
   type: NodePort
   ports:
   - name: web
-    nodePort: 30903
+    # nodePort: 30903
```

Comment:
I *think* the `nodePort` can be added back in. I commented them out when I was testing across multiple namespaces